src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -176,7 +186,11 @@ function startWatchingForValueChange(target, targetID) {
   );
 
   Object.defineProperty(activeElement, 'value', newValueProp);
-  activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  if (activeElement.attachEvent) {
```

Comment:
@jquense I have access to IE8-11, but yeah, if this works and was this way before, let's just keep it that way.